transformer/assembly/__tests__/file2ByteArray.spec.ts

```diff
@@ -0,0 +1,12 @@
+export function strToStaticArray(str: string): StaticArray<u8> {
```

Comment:
Is there a need to export this function?

```suggestion
function strToStaticArray(str: string): StaticArray<u8> {
```